ts/components/screens/BaseScreenComponent.tsx

```diff
@@ -16,11 +16,21 @@ interface OwnProps {
   contextualHelp?: ContextualHelpProps;
   headerBody?: React.ReactNode;
   appLogo?: boolean;
+  dark?: boolean;
   isSearchAvailable?: boolean;
   searchType?: SearchType;
 }
 
-type BaseHeaderProps = "appLogo" | "primary" | "goBack" | "headerTitle";
+type BaseHeaderProps =
```

Comment:
Already included in the OwnProps interface, please remove the duplicates.

Reply:
done!